scripts/sct_extract_metric.py

```diff
@@ -887,9 +944,10 @@ def save_metrics(labels_id_user, indiv_labels_ids, combined_labels_ids, indiv_la
 
     sct.printv('\tDone.')
 
-
-def check_method(method, fname_normalizing_label, normalization_method):
-    """Check the consistency of the methods asked by the user."""
+#=======================================================================================================================
```

Comment:
do we want to go with this convention for every sub-function, instead of using """?